queries.sql

```diff
@@ -1,10 +1,48 @@
 -- Queries implemented on animals table
-
 SELECT * FROM animals WHERE name like '%mon';
 SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-12-31';
 SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
 SELECT date_of_birth FROM animals where name LIKE '%Agumon' OR name LIKE '%Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name NOT LIKE '%Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Transaction queries
+BEGIN TRANSACTION;
+UPDATE animals SET species='unspecified';
+Select * FROM animals;
+ROLLBACK TRANSACTION;
+SELECT * FROM animals;
+
+BEGIN WORK;
+UPDATE animal SET species = 'digimon' WHERE name LIKE '@mon';
+UPDATE animal SET species = 'pokemon' WHERE name NOT LIKE '@mon';
+SELECT * FROM animals;
+COMMIT;
```

Comment:
- [ ] Dear @mirwaisfarahi, To verify that change persists after commit, Please add a select statement next to the `COMMIT` statement.

Reply:
Dear @TadesseDev , Thank you for the feedback. The statement is added.